map/manage-bookmarks/README.md

```diff
@@ -1,37 +1,30 @@
-# Manage Bookmarks
+# Manage bookmarks
 
-Use bookmarks in a map.
+Access and create bookmarks on a map.
 
-![](ManageBookmarks.png)
+![Image of manage bookmarks](ManageBookmarks.png)
 
-## How to use the sample
+## Use case
 
-The map in the sample comes pre-populated with a set of bookmarks.
-  To access a bookmark and move to that location.
-  - click on a bookmark's name from the list
+Bookmarks are used for easily storing and accessing saved locations on the map. Bookmarks are of interest in educational apps (e.g. touring historical sites) or more specifically, for a land management company wishing to visually monitor flood levels over time at a particular location. These locations can be saved as bookmarks and revisited easily each time their basemap data has been updated (e.g. working with up to date satellite imagery to monitor water levels).
```

Comment:
I think one use case is good enough. A mention of allowing users to add their own bookmarks would also be good.

```suggestion
Bookmarks are saved extents for a map. A tour app may save a bookmark for each site in the tour. An app may also allow users to add their own favorite places as bookmarks.
```